### PR TITLE
feat: improve drive_contribute command, closes #194, #192

### LINKED ISSUE #192: Aggiornare comando "☁️  Cloud"
il comando ☁️  Cloud (o /cloud) mostra DMI Cloud, sito web ormai archiviato, andrebbe tolto da li'

Inoltre, non spiega come funziona il comando /drive_contribute per contribuire alla cartella drive.

### LINKED ISSUE #194: Drive contribute errors
Gestire gli errori nel caso in cui l'email non esiste o e' mal formata.

```
An exception was raised:
  File "/dmibot/module/commands/aulario.py", line 106, in calendar_handler
    context.bot.deleteMessage(chat_id=chat_id, message_id=query.message.message_id)
telegram.error.BadRequest: Message can't be deleted for everyone
```

```
An exception was raised:
  File "/dmibot/module/commands/drive_contribute.py", line 44, in drive_contribute
    permission = drive_root_folder.InsertPermission(
pydrive2.files.ApiRequestError: <HttpError 400 when requesting https://www.googleapis.com/drive/v2/files/TOKEN/permissions?supportsAllDrives=true&alt=json returned "Invalid permission value". Details: "[{'domain': 'global', 'reason': 'invalid', 'message': 'Invalid permission value', 'locationType': 'other', 'location': 'permission.value'}]">
```

### PR DESCRIPTION
**Pull request by Helias**
closes #194
closes #192

### CODE REVIEW

File: module/commands/drive_contribute.py [hunk 1]
```diff
@@ -27,30 +27,42 @@ def drive_contribute(update: Update, context: CallbackContext):
...
+        )
+
+        context.bot.sendMessage(
+            chat_id=update.message.chat_id,
+            text=f"Hai ottenuto l'accesso in scrittura alla cartella Drive! \n\nPresto ti arriverà un'email di conferma per gli accessi in scrittura e potrai aggiungere appunti nella cartella mediante questo link https://cutt.ly/unict-dmi-drive",
+        )
+    except:
+        context.bot.sendMessage(
+            chat_id=update.message.chat_id,
+            text=f"Si é verificato un errore durante la validazione dell'email, riprova piú tardi o verifica se hai già gli accessi in scrittura alla cartella mediante questo link https://cutt.ly/unict-dmi-drive",
```
Comment (dcariotti) [inline module/commands/drive_contribute.py:66]: What is `ú`? Can you fix this typo pls?

```suggestion
            text=f"Si é verificato un errore durante la validazione dell'email, riprova più tardi o verifica se hai già gli accessi in scrittura alla cartella mediante questo link https://cutt.ly/unict-dmi-drive",
```

File: module/commands/drive_contribute.py [hunk 2]
```diff
@@ -27,30 +27,42 @@ def drive_contribute(update: Update, context: CallbackContext):
...
+                "permission_obj": permission,
+                "request_message": request_message,
+            },
+        )
+
+        context.bot.sendMessage(
+            chat_id=update.message.chat_id,
+            text=f"Hai ottenuto l'accesso in scrittura alla cartella Drive! \n\nPresto ti arriverà un'email di conferma per gli accessi in scrittura e potrai aggiungere appunti nella cartella mediante questo link https://cutt.ly/unict-dmi-drive",
+        )
+    except:
```
Comment (dcariotti) [inline module/commands/drive_contribute.py:63]: Can you add the handled exception class(es) pls? this way we could see if there are any other bug(s)

**Review by dcariotti — CHANGES_REQUESTED** [review]

File: module/commands/drive_contribute.py (hunk 2 above)
Comment (Helias) [reply module/commands/drive_contribute.py]: done

**Review by dcariotti — APPROVED** [review]
LGTM

**Pull request merged**